app/src/main/java/com/simplecity/amp_library/ui/screens/artist/menu/AlbumArtistMenuPresenter.kt

```diff
@@ -129,8 +136,13 @@ class AlbumArtistMenuPresenter @Inject constructor(
         const val TAG = "AlbumMenuContract"
     }
 
-    fun newUiEvent(uiEventType: UiEventType, albumArtist: AlbumArtist){
+    private fun newUiAlbumArtistEvent(uiEventType: UiEventType, albumArtist: AlbumArtist){
```

Comment:
For methods that are only used for one event type, I'd remove that event type from the method signature to avoid mistakes where you accidentally pass in the wrong event type. And then within the method you can set the event type enum.